style.css

```diff
@@ -3658,7 +3661,7 @@ body.resource #slider-container {
content-wrap .button a:after, .about-banner #abt-banner-container .banner-wrap .content-wrap .about-hero #about-hero_cmp .aem-Grid .about-hero-heading-group #about-hero-heading-group_cmp .button a:after, .about-hero #about-hero_cmp .aem-Grid .about-hero-heading-group #about-hero-heading-group_cmp .about-banner #abt-banner-container .banner-wrap .content-wrap .button a:before, .about-banner #abt-banner-container .banner-wrap .content-wrap .about-hero #about-hero_cmp .aem-Grid .about-hero-heading-group #about-hero-heading-group_cmp .button a:before, .about-hero #about-hero_cmp .aem-Grid .about-hero-heading-group #about-hero-heading-group_cmp .about-banner #abt-banner-container .bottom-container #bottom-wrap .bottom-content .button a, .about-banner #abt-banner-container .bottom-container #bottom-wrap .bottom-content .about-hero #about-hero_cmp .aem-Grid .about-hero-heading-group #about-hero-heading-group_cmp .button a, .about-hero #about-hero_cmp .aem-Grid .about-hero-heading-group #about-hero-heading-group_cmp .about-banner #abt-banner-container .bottom-container #bottom-wrap .bottom-content .button a:after, .about-banner #abt-banner-container .bottom-container #bottom-wrap .bottom-content .about-hero #about-hero_cmp .aem-Grid .about-hero-heading-group #about-hero-heading-group_cmp .button a:after, .about-hero #about-hero_cmp .aem-Grid .about-hero-heading-group #about-hero-heading-group_cmp .about-banner #abt-banner-container .bottom-container #bottom-wrap .bottom-content .button a:before, .about-banner #abt-banner-container .bottom-container #bottom-wrap .bottom-content .about-hero #about-hero_cmp .aem-Grid .about-hero-heading-group #about-hero-heading-group_cmp .button a:before, .about-hero #about-hero_cmp .aem-Grid .about-hero-heading-group #about-hero-heading-group_cmp .about-banner #abt-banner-container .bottom-container #bottom-joingroup > div:first-child .bottom-content .button a, .about-banner #abt-banner-container .bottom-container #bottom-joingroup > div:first-child .bottom-content .about-hero #about-hero_cmp .aem-Grid .about-hero-heading-group #about-hero-heading-group_cmp .button a, .about-hero #about-hero_cmp .aem-Grid .about-hero-heading-group #about-hero-heading-group_cmp .about-banner #abt-banner-container .bottom-container #bottom-joingroup > div:first-child .bottom-content .button a:after, .about-banner #abt-banner-container .bottom-container #bottom-joingroup > div:first-child .bottom-content .about-hero #about-hero_cmp .aem-Grid .about-hero-heading-group #about-hero-heading-group_cmp .button a:after, .about-hero #about-hero_cmp .aem-Grid .about-hero-heading-group #about-hero-heading-group_cmp .about-banner #abt-banner-container .bottom-container #bottom-joingroup > div:first-child .bottom-content .button a:before, .about-banner #abt-banner-container .bottom-container #bottom-joingroup > div:first-child .bottom-content .about-hero #about-hero_cmp .aem-Grid .about-hero-heading-group #about-hero-heading-group_cmp .button a:before {
```

Comment:
@Prystal254 I was looking over your compiled CSS file and found several giant rulesets (like this one) that shouldn't have happened when compared to the SCSS. For example: `.about-hero #about-hero_cmp .aem-Grid .about-hero-heading-group #about-hero-heading-group_cmp .bottom-row` shouldn't be possible. I believe compilation is getting messed up due to your use of the `%transition` placeholder selectors. I've never seen them defined inside a ruleset before. Even the [SCSS documentation](https://sass-lang.com/documentation/style-rules/placeholder-selectors) doesn't show that. Since all three instances of `%transition` equal `transition: .12s`, you'd be better off using a variable and referencing it like `transition: $transitionDuration`.